drafts/swift/redux-in-swift-pt-1.md

```diff
@@ -0,0 +1,280 @@
+# Redux in Swift Part 1: Introduction and Example within 70 Lines!
+
+In this blog, I will introduce you to the concept of Redux architecture briefly, implement and explain it in Swift bit by bit with minimum changes each time, while satisfying Redux's requirements and making it compilable at the same time.
+
+## Redux 
+
+> Redux architecture revolves around a **strict unidirectional data flow**.
+
+![](https://www.dropbox.com/s/51evjhxw35t59x2/Redux.png?raw=1)
+
+[Redux Doc: Data Flow](http://redux.js.org/docs/basics/DataFlow.html)
+
+### WAT?
+
+In essense, it's an idea which trys to
+
+- seperate/isolate where changes can happen to the "State" of an application
```

Comment:
I understand what you mean by this line, but I wonder if it could be made clearer: would this communicate the same thing?
> seperate/isolate the 'state' of an application to a single location